src/tests/ftest/control/dmg_system_cleanup.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from avocado.core.exceptions import TestFail
+from pydaos.raw import (DaosApiError, DaosPool)
+from general_utils import DaosTestError
+from apricot import TestWithServers
+
+
+class DmgSystemCleanupTest(TestWithServers):
+    """Test Class Description:
+    This test covers the following requirement.
+    (SRS-326) Pool Management - automatic pool handle revocation
+
+    Test step:
+    1. Create 2 pools.
+    2. Create a container in each pool.
+    3. Write to each container to ensure connections are working.
+    4. Call dmg system cleanup host.
+    5. Write to each container again to ensure they fail
+    6. Check that cleaned up handles match expected counts.
+
+    :avocado: recursive
+    """
+
+    def test_dmg_system_cleanup_one_host(self):
+        """
+        JIRA ID: DAOS-6471
+
+        Test Description: Test dmg system cleanup.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=small,dmg
+        :avocado: tags=control,dmg_system_cleanup
+        """
+        # Create 2 pools and create a container in each pool.
+        self.pool = []
+        self.container = []
+        for _ in range(2):
+            self.pool.append(self.get_pool())
+            self.container.append(self.get_container(self.pool[-1]))
+
+        # Create 5 more connections to each pool
+        pool_handles = []
+        for pool in self.pool:
+            for _ in range(5):
+                handle = self.get_pool(create=False, connect=False)
+                handle.pool = DaosPool(self.context)
+                handle.uuid = pool.uuid
+                handle.connect(2)
+                pool_handles.append(handle)
+
+        # Check to make sure we can access the pool
+        try:
+            for i in range(2):
+                self.container[i].get_params(self)
```

Comment:
```suggestion
```
You used self.get_container, so get_params was already called for you.

Reply:
Ok I wasn't sure if this was needed but I saw it used in the evict test so I added it here.